.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: If you have a feature request or an idea that you believe could improve our
+  software, we would love to hear from you.
+title: "[Feature]:"
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
I would vote for leaving out the example

Reply:
ok